lib/tasks/update_database_records_in_big_query.rake

```diff
@@ -0,0 +1,8 @@
+namespace :database_records do
```

Comment:
I think I'd prefer namespace `:big_query`, then task `:export`.  I'm not sure what the `database_records` adds.